lib/utils/checkConfigExists/index.js

```diff
@@ -0,0 +1,18 @@
+const { access, constants } = require('fs');
+const { promisify } = require('util');
+const accessAsync = promisify(access);
+
+async function checkConfigExists() {
+    try {
+        const configFileName = `${__dirname}/../../../.config`;
```

Comment:
Since this is the third time we're getting this path maybe we should put it somewhere else.
I'm thinking a function `getConfigPath()` would do good here.
Then also `readConfig` and `writeConfig` should be updated.

Reply:
Agreed, I was thinking the same. Maybe even actually have a single config module with wrte/read and check in the same file / folder.